File: tournaments/company-bot-strategy/company-bot-strategy.py
def companyBotStrategy(trainingData):
    correctAnswers=[trainingData[i][0] for i in range(len(trainingData)) if trainingData[i][1]==1]
    if len(correctAnswers)!=0: 
        return sum(correctAnswers)/len(correctAnswers)
    else: return 0

File: tournaments/company-bot-strategy/company-bot-strategy.js
function companyBotStrategy(trainingData) {
    let time = 0;
    let correctness = 0;
    trainingData.forEach(data => {
      if (data[1] === 1) {
        time += data[0];
        correctness += data[1];
      }
    });
  
    return time / correctness || 0;
}
  
console.log(
    companyBotStrategy([
      [3, 1],
      [6, 1],
      [4, 1],
      [5, 1]
    ])
);

console.log(
    companyBotStrategy([
      [4, 1],
      [4, -1],
      [0, 0],
      [4, 1]
    ])
);